model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -48,7 +48,15 @@ data class AnalyzerConfiguration(
      * Package manager specific configurations. The key needs to match the name of the package manager class, e.g.
      * "NuGet" for the NuGet package manager.
      */
-    val packageManagers: Map<String, PackageManagerConfiguration>? = null
+    val packageManagers: Map<String, PackageManagerConfiguration>? = null,
+
+    /**
+     * A flag to control whether excluded scopes should be skipped when constructing the dependency graph. If set to
+     * true, only the dependencies from non-excluded scopes are added to the dependency graph. This is useful for
+     * projects with large dependency graphs, for instance Gradle Android projects. By setting this flag to true,
+     * the size of the graph and the analysis time can be reduced significantly.
+     */
+    val skipExcludedScopesInDependencyGraph: Boolean = false
```

Comment:
A dependency can be excluded in two ways:

1. path exclude for the corresponding project
2. scope exclude (s)

So, I believe there should be a single flag to toggle inclusion of "excluded dependencies"
So, how about naming this just e.g.: 
- skipExcludedDependencies
- skipExcludedPackages

What do you think?

Apart from that, I'd prefer to drop the mention of "Dependency graph" as that's IMO not much relevant, just "dependencies" or "package" IMO should be enough.